contracts/OctobayGovTokenFactory.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import './OctobayStorage.sol';
+import './OctobayGovToken.sol';
+
+contract OctobayGovTokenFactory is OctobayStorage {
+
+    event NewTokenEvent(string name, string symbol, address tokenAddr);
+    event UpdatedProjectId(string oldProjectId, string newProjectId);
+    mapping (string => OctobayGovToken) public tokensByProjectId;
+
+    /// @param _name Name of the new token
+    /// @param _symbol Token Symbol for the new token
+    /// @param _projectId Path of the org or repo which maps to the new token
+    /// @return The address of the new token contract
+    function createToken(string memory _name, string memory _symbol, string memory _projectId) external onlyOctobay returns (OctobayGovToken) {
```

Comment:
needs to implemention of those parameters mentioned above then (https://github.com/Octobay/contracts/pull/9/files#r600722923)

Reply:
and they need to be updateable by owner (and later also by the owner-nft holder)